src/hooks/use-list-behavior.js

```diff
@@ -0,0 +1,50 @@
+import { useCallback } from 'react'
+import safeInvoke from '../lib/safe-invoke'
+import { useRovingTabindex } from '../roving-tabindex'
+import { useClickable } from './use-clickable'
+
+export function useListBehavior({
+  disabled,
```

Comment:
What happens here if `disabled` is undefined? Should any of these props have default values?